src/main/java/org/elasticsearch/index/query/FQueryFilterBuilder.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.index.query;
+
+/**
+ * A filter that simply wraps a query. This extension of the QueryFilterBuilder only differs in the NAME constant
+ * that links it to a separate parser.
+ * @deprecated Useless now that queries and filters are merged: pass the
+ *             query as a filter directly.
+ */
+@Deprecated
+public class FQueryFilterBuilder extends QueryFilterBuilder {
+
+    public static final String NAME = "fquery";
+
+    public FQueryFilterBuilder(QueryBuilder queryBuilder) {
+        super(queryBuilder);
+    }
+
```

Comment:
The QueryFilterBuilder is going to be interesting when we will try and address #10776 as it doesn't support query name actually but f query filter does.... this is confusing to me but we can't do much about it right now I guess. I think I would do a proper separation of FQueryFilterBuilder and QueryFilterBuilder though. let's have this 1:1 with the rest layer, that means moving the query name here and splitting the current toXContent into two, one that generates query filter and one to be moved here that generates fquery.